chef/cookbooks/network/recipes/default.rb

```diff
@@ -393,6 +393,20 @@ def get_datapath_id_for_ovsbridge(bridge)
     nic.tx_offloading = node["network"]["enable_tx_offloading"] || true
   end
 
+  # if the nic is a vlan and the physical parent is not used, also set
+  # the mtu from the vlan nic for the parent
+  if nic.kind_of?(Nic::Vlan)
+    unless ifs[nic.parent].key? "mtu"
```

Comment:
Shouldn't we ignore the mtu setting if it's lower than the one for the vlan interface?